lnwallet/channel.go

```diff
@@ -5489,18 +5295,15 @@ func (lc *LightningChannel) ReceiveRevocation(revMsg *lnwire.RevokeAndAck) (
 	localChainTail := lc.commitChains.Local.tail().height
 
 	source := lc.ShortChanID()
-	chanID := lnwire.NewChanIDFromOutPoint(lc.channelState.FundingOutpoint)
 
 	// Determine the set of htlcs that can be forwarded as a result of
 	// having received the revocation. We will simultaneously construct the
 	// log updates and payment descriptors, allowing us to persist the log
 	// updates to disk and optimistically buffer the forwarding package in
 	// memory.
 	var (
-		addsToForward        []*PaymentDescriptor
-		addUpdates           []channeldb.LogUpdate
-		settleFailsToForward []*PaymentDescriptor
-		settleFailUpdates    []channeldb.LogUpdate
+		addUpdates        []channeldb.LogUpdate
```

Comment:
I think we should remove these two and keep the other two.

Reply:
Do you mean just rename them? The idea is we specifically don't want to leak the payment descriptors.

Reply:
No what I meant was to keep `addsToForward` and `settleFailsToForward` and use these in `channeldb.NewFwdPkg` to create the fwd pkg instead of `addUpdates` or `settleFailUpdates` because we want the filtered log updates inside the fwd pkg - this also matches the current behavior, as the existing `ReceiveRevocation` returns the `addsToForward`, and after this PR it would be referenced from `fwdPkg.Add`.

Reply:
Got it, so still use LogUpdates, but use the filtering conditions we currently apply to `addsToForward`